libs/libvtrutil/src/vtr_dynamic_bitset.h

```diff
@@ -0,0 +1,54 @@
+#ifndef VTR_DYNAMIC_BITSET
+#define VTR_DYNAMIC_BITSET
+
+#include <limits>
+#include <vector>
+
+namespace vtr {
+
+template<typename Storage = unsigned int>
+class dynamic_bitset {
```

Comment:
Also, what do you think about:
```cpp
template<typename Index, typename Storage = unsigned int>
class dynamic_bitset {
...
void get(Index index);
void set(Index index, bool val);
```

Which would allow the index type to be explicitly specified:
```cpp
vtr::dynamic_bitset<ClusterNetId> is_routed;
```
this would match the behaviour of other VTR containers (like vtr::vector), enforcing type checking and preventing mistakes like passing a ClusterNetId where an AtomNetId is expected.

Reply:
Fixed